### ISSUE TITLE
Houdini DSO Error 18.5

### ISSUE THREAD

**Issue by van-8** [elrnv/vtkio#11]
Hi! I'm a novice to Houdini & programming in general, trying to learn to create motion design from neuroscience data. Thanks so much for creating this tool!! 

I receive this error after creating new folder called `dso` and unzipping contents into it. I can't find a cargo.toml file to modify, so perhaps that's my problem? Any help would be appreciated. Cheers!

<img width="306" alt="vtkio" src="https://user-images.githubusercontent.com/28463475/110136846-24149200-7e0b-11eb-8535-f2c18ea9ac54.PNG">

Windows 10 (20H2)
Houdini 18.5.462 Apprentice Edition

**Comment by elrnv** [comment]
Thanks for the feedback!
Are you trying to install the vtkio lib from https://elrnv.com/software/houdini-plugins/ ?
If so, I will have time today to shelf out a more recent build. The current build vtkio v0.1.0 is for an earlier version of Houdini.

Just a note, that this repo is for a library that the Houdini plugin uses, not the Houdini plugin itself (which unfortunately has the same name). I am currently working on a repo for the vtkio plugin, which should have instructions about building the plugin.

**Comment by van-8** [comment]
Yes, trying to install from that url. Really appreciate the fantastic tool, take your time! 

Once you're happy with it, perhaps I can make a video tutorial on install/using it? I'm a filmmaker by training, expanding my skillset into synthetic art. Here's something I put together for [JupyterLab](https://www.youtube.com/watch?v=A5YyoCKxEOU)

**Comment by elrnv** [comment]
Oh wow that is a really nice video! I use JupyterLab all the time :)
I would be eternally grateful if you decided to make a video about this lib, but I suspect there is not much to it than just dropping some files in a folder, and opening up a vtk/vtu/vtp file in Houdini.

Anyways I updated the page [plugins page](https://elrnv.com/software/houdini-plugins/) with the new 0.2.0 versions of the vtkio plugin for Houdini so hopefully it will work for you now, but please let me know if it doesn't.

As for actually building the code, currently the plugin is located as an example in https://github.com/elrnv/hdkrs but I will be moving it to its own repo at some point. The build process there is currently under construction, and I'm not even sure what the build instructions will look like when it's all done. I am honestly not sure when I will get a chance to wrap it all up though. I will keep this issue open until at least the build instructions for the Houdini plugin are complete, in case someone else finds it.

**Comment by van-8** [comment]
So fast! And worked perfectly!

Since you're pioneering this new sub-discipline of using vtk's in Houdini, do you have any advice on preparing the file for use? I've imported, but lost on how to work with the data. This is another reason I'd love to make a video: guiding others through any necessary steps so they can increase their understanding of the software & the format. 

Personally, I'm trying to reconstruct [white matter tracts](http://brainspan.org/api/v2/well_known_file_download/157780825) in the human brain using data from the [Allen Institute](http://brainspan.org/static/download.html).  I'm reading up on the vtk.org docs but would love any advice.

**Comment by elrnv** [comment]
Oh this is a great real-world example! Thank you! It looks like that data set consists of just points. Currently the HDK plugin is focused on polygon meshes and tet meshes, although it is probably not uncommon to have points without cells in real world data. It looks like points are not properly loaded at the moment. I will look into this and get back to you soon.

**Comment by elrnv** [comment]
Just wanted to leave an update here
It seems like that referenced zip failed to unzip properly on my machine and so the first file I was looking at was truncated and only had the points section. The other VTK files are valid and contain polylines.

While this lib (vtkio) supports all vtk formats, the VTK Houdini plugin does not yet support polylines so you can't load a polyline vtk directly into Houdini yet. Having said that, ParaView already supports exporting to Houdini .geo format, so for the time being you can open up a Vtk file in ParaView and export it to .geo to be loaded in Houini.

In the meantime I will work on adding support for point clouds and polylines to the Houdini plugin :)
